File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/nursing/shared/dto/nursing-opd-checkin.dto.ts

import { ChiefComplaints_DTO } from "./chief-complaints.dto";
import { FinalDiagnosis_DTO } from "./final-diagnosis.dto";

export class NursingOpdCheckIn_DTO {
    public PatientVisitId: number = 0;
    public PatientId: number = 0;
    public PerformerId: number = null;
    public PerformerName: string = null;
    public DiagnosisList: Array<FinalDiagnosis_DTO> = [];
    public ChiefComplaints: Array<ChiefComplaints_DTO> = [];

}

File: Code/Websites/DanpheEMR/Services/Nursing/DTOs/NursingOpdCheckIn_DTO.cs
using DanpheEMR.ServerModel;
using DanpheEMR.ServerModel.ClinicalModels;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DanpheEMR.Controllers.Nursing.DTOs
{
    public class NursingOpdCheckIn_DTO
    {
        public int PatientVisitId { get; set; }
        public int PatientId { get; set; }
        public int? PerformerId { get; set; }
        public string PerformerName { get; set; }

        public List<AddFinalDiagnosis_DTO> DiagnosisList { get; set; }

        public List<AddChiefComplaint_DTO> ChiefComplaints { get; set; }


    }
}
